pkg/odo/cli/component/create.go

```diff
@@ -206,7 +208,7 @@ A full list of component types that can be deployed is available using: 'odo cat
 			odoutil.CheckError(err, "")
 
 			// Git is the only one using BuildConfig since we need to retrieve the git
-			err = component.Build(client, componentName, applicationName, true, stdout)
+			err = component.Build(client, componentName, applicationName, componentWait, stdout)
```

Comment:
can you use "wait" as a variable name instead of "componentWait"? Makes more sense.